src/sentry/static/sentry/app/stores/groupStore.jsx

```diff
@@ -16,15 +16,13 @@ const GroupStore = Reflux.createStore({
   listenables: [GroupActions],
 
   init() {
-    this.items = [];
-    this.statuses = {};
-    this.pendingChanges = new utils.PendingChangeQueue();
+    this.reset();
   },
 
   reset() {
     this.items = [];
     this.statuses = {};
-    this.pendingChanges.clear();
```

Comment:
Not sure I follow? I just replaced this line w/ just re-instantiating a new `PendingChangeQueue`.